demo/src/components/MiniCart/MiniCart.js

```diff
@@ -0,0 +1,60 @@
+import {Container, Col, Row} from 'react-bootstrap';
+import {connect} from 'react-redux';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {computePrice} from '../Cart/Cart.js';
+import '../Cart/Cart.css';
+
+/**
+ * Creates a component describing a shopping cart.
+ * @param {!Object<string,
+ *      {name:string, item:!Object, quantity:number, description:string,
```

Comment:
Likewise, if this item type is used across several modules, consider which it would be best to define it in, or even create a new "types" module for holding types used around the site.

Reply:
I've added a typedef in store/store.js since that's where the object is defined. No module since I don't think we will have any other big types.